backend/tickets/models/tickets_event_meta.py

```diff
@@ -134,13 +133,7 @@ class TicketsEventMeta(ContactEmailMixin, EventMetaBase, LocalizedModel):
         related_name="as_accommodation_access_group_for",
     )
 
-    terms_and_conditions_url = LocalizedCharField(
-        blank=True,
-        default=dict,
-        max_length=1023,
-        verbose_name=_("Terms and conditions URL"),
-        help_text=_("If set, customers will be required to indicate acceptance to finish order."),
-    )
+    terms_and_conditions_url = HStoreField(blank=True, default=dict)
```

Comment:
Ditto for help text here. Also, wonder what the semantics for the default are - is it a default for each key..?

Reply:
No, it's a default for the whole field.